web/src/tippyjs.ts

```diff
@@ -178,6 +182,88 @@ export function initialize(): void {
         placement: "right",
         delay: EXTRA_LONG_HOVER_DELAY,
         appendTo: () => document.body,
+        onShow(instance) {
+            const $container = $(instance.popper).find(".views-tooltip-container");
+            let display_count = 0;
+            const sidebar_option = $container.attr("data-view-code");
+
+            switch (sidebar_option) {
+                case user_settings.web_home_view:
+                    $container.find(".views-tooltip-home-view-note").removeClass("hide");
+                    display_count = unread.get_unread_message_count();
+                    $container.find(".views-message-count").text(
+                        $t(
+                            {
+                                defaultMessage:
+                                    "{display_count, plural, one {You have {display_count} unread message.} other {You have {display_count} unread messages.}}",
```

Comment:
I think we can also include a "=0" case here, saying "You have no unread messages.", instead of saying "You have 0 unread messages.". Naturally, that would also apply to all the other tooltips in this PR.

Reply:
Yeah this sounds better